File: benchflow-dsl/src/main/scala/cloud/benchflow/dsl/definition/sut/suttype/SutType.scala
package cloud.benchflow.dsl.definition.sut.suttype

/**
 * @author Jesper Findahl (jesper.findahl@usi.ch)
 *         created on 2017-05-22
 */

object SutType extends Enumeration {
  type SutType = Value
  val WfMS = Value("wfms")
  val Http = Value("http")
}

File: benchflow-dsl/src/main/scala/cloud/benchflow/dsl/definition/sut/suttype/SutType.java
package cloud.benchflow.dsl.definition.sut.suttype;

/**
 * @author Jesper Findahl (jesper.findahl@gmail.com)
 *         created on 2017-06-23
 */
public enum SutType {
  WFMS("wfms"),
  HTTP("http")
  ;

  private final String stringValue;

  SutType(String stringValue) {
    this.stringValue = stringValue;
  }

  @Override
  public String toString() {
    return stringValue;
  }
}
